### ISSUE TITLE
Add th:attrmap for setting several attributes at a time from a Map

### ISSUE THREAD

**Issue by AaricChenLegacy** [thymeleaf/thymeleaf#687]
e.g.

```
Map<String,String> attrMap = new Map();

attrMap .put("data-key1","11111");
attrMap .put("data-key2","2222");

<p th:attr="${attrMap}"></p>
```
and the render result will be:
```
<p data-key1="1111" data-key2="2222"></p>
```

it's support this feature ?

**Comment by danielfernandez** [comment]
No, I'm afraid there is no way to automatically add a set of attributes from a map like you describe. I'm going to leave this open though as a suggestion for future versions. Maybe a `th:attrmap` attribute could be useful if no technical issues are found with this.

**Comment by AaricChenLegacy** [comment]
that's would be best if thymeleaf can have this feature.
hope we can get it ASAP.

**Comment by wimdeblauwe** [comment]
I could use this. I have written a `AbstractAttributeTagProcessor` that does this now. But it would be great to have this out of the box from Thymeleaf.
